src/TeamSpeak3QueryApi/Notifications/ClientMoved.cs

```diff
@@ -1,4 +1,4 @@
-namespace TeamSpeak3QueryApi.Net.Specialized.Notifications
+namespace TeamSpeak3QueryApi.Net.Notifications
```

Comment:
The `Specialized` contained everything related to the rich client (not the query client). That separation made sense to me.

Reply:
Ah okey. But doesn't it make more sense to put the query connection to separate folder? So we have the primary class which will be used in the root folder

Reply:
See other comments on this.